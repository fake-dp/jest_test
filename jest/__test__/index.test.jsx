import JestUnitTestPage from "../pages/jest/unittest/index";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";

it("내가 원하는대로 그려지는지 test", () => {
  render(<JestUnitTestPage />);

  const myText = screen.getByText("바바는 고양이입니다.");
  expect(myText).toBeInTheDocument();

  const myText2 = screen.getByText("바바의 취미 :");
  expect(myText2).toBeInTheDocument();

  const myText3 = screen.getByText("클릭미");
  expect(myText3).toBeInTheDocument();
});
