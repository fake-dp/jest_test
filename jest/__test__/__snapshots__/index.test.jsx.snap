// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`내가 원하는대로 그려지는지 test 1`] = `
<div>
  <div>
    바바는 고양이입니다.
  </div>
  바바의 취미 : 
  <input
    type="text"
  />
  <button>
    클릭미
  </button>
</div>
`;
